.github/workflows/ci.yml

```
@@ -68,7 +68,7 @@ jobs:
 
       - uses: julia-actions/julia-processcoverage@v1
 
-      - uses: codecov/codecov-action@v3
+      - uses: codecov/codecov-action@v5
         with:
           file: lcov.info
```

```suggestion
          files: lcov.info
```
and an admin needs to create a token

---

Yep, I see a CODECOV_TOKEN in the GitHub org settings for this org, so we should be good to go